sound/soc/sof/ops.h

```diff
@@ -37,6 +37,14 @@ static inline int snd_sof_remove(struct snd_sof_dev *sdev)
 	return 0;
 }
 
+static inline int snd_sof_shutdown(struct snd_sof_dev *sdev)
```

Comment:
does shutdown need to return int?

Reply:
> does shutdown need to return int?

the sof_pci_shutdown() itself doesn't require int return, here returning int with snd_sof_shutdown() to make it consistent with other snd_sof_xxx()s. This is exactly same with what we have for snd_sof_device_remove() and sof_pci_remove().